### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by 532910** [raphaelm/android-barcode-keyboard#1]
Add it to F-Droid please.
https://gitlab.com/fdroid/rfp/-/issues/1462

**Comment by IzzySoft** [comment]
To speed things app: if you'd attach signed APKs to your tags/releases, I could add them to [my repo](https://apt.izzysoft.de/fdroid) immediately (and later remove them again from there when your app got listed in F-Droid with working updates).

Thanks to @532910 opening an RFP, inclusion process with F-Droid got started already (would be nice if you could confirm this is welcome) and usually takes something starting at 1..2 weeks (if no issues show up). F-Droid would welcome if you could add [Fastlane structures](https://gitlab.com/snippets/1895688) to your app's repo. Apart from that, everything should be fine (as the F-Droid IssueBot will hopefully confirm in a few hours).

**Comment by raphaelm** [comment]
Yes, inclusion process on F-Droid is welcome!

I created the signed release on a different computer yesterday which I don't have with me right now, so not sure if I will remember to put up some APKs, but if I do, I'll let you know ;)

Thanks for letting me know about the Fastlane structure. I'm not sure I'll have the time to put in the effort at the moment, but I'll keep it in mind :)

**Comment by IzzySoft** [comment]
Thanks Raphael! I've just ticked the corresponding check-box in @532910's RFP – which (after "bot scan & confirmation") will wait for your providing Fastlane (should be fairly easy and take not more than 15min for an "initial setup" – the very basic would be providing `short_description.txt` and `full_description.txt` for `en-US`; with that, we could start at F-Droid and you could add screenshots etc. with future releases).

And yes, please ping me here if the APK is available!

**Comment by IzzySoft** [comment]
Please also drop a note here (or at the RFP) once Fastlane is available, so I can check and further process the RFP.

**Comment by raphaelm** [comment]
I will likely not be able to spend any more time on this project other than initially creating it, sorry :(

**Comment by IzzySoft** [comment]
Sad to read! So shall we cancel the RFP then?

**Comment by raphaelm** [comment]
Well, of course anyone else could write a description just as well, if someone is interested, I'd certainly merge PRs, but yeah, I probably wont do it, this was a "solve the problem I have then dump it somewhere in case others need it" kind of project, not something I can make room for actively working on.

**Comment by IzzySoft** [comment]
Yes, sure – and it's great you didn't just dump it when no longer needed but made it available! If someone picks it up and maintains it, that would be really great. But we hesitate adopting "unmaintained projects". If you'd just attach the APK you've built in August (must have "slipped through" somehow), folks could also take that from here. Repos with automated updates are good if there are updates :wink:

But yes, if someone provides the description etc I can flag the RFP accordingly and see what the packagers say.

**Comment by bene64** [comment]
I've just closed the rfp issue to reduce the number of open issues at the F-Droid side.
